src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -52,6 +63,25 @@
            [:div {} k [:span {:style {:margin "0 1em"}} "→"] (display-value v)])])])})
 
 
+(react/defc WorkflowTiming
+  {:get-initial-state
+   (fn []
+     {:expanded false})
+   :render
+   (fn [{:keys [props state]}]
+     [:div {}
+      (create-field
+        (:label props)
+        (if (empty? (:data props))
+          "Not Available"
+          (style/create-link {:text (if (:expanded @state) "Hide" "Show")
+                              :onClick #(swap! state assoc :expanded (not (:expanded @state)))})))
+          [:div {:style {:padding "0.25em 0 0 0"} :id "chart_div"}]
+        (if (:expanded @state)
+          (draw-chart (:data props) (:workflow-name props))
+          (clear-chart))
```

Comment:
This is doing a mutation in a render function, which seems suspect.  I'm not sure exactly how this works, but can this just be

    (when (:expanded @state)
      (.timingDiagram js/window data workflow-name))

I also don't see `timingDiagramDrawn` being used anywhere, other than to guard `clear-chart` which seems unnecessary.